apps/dsb-message-broker/src/message/error.handler.ts

```diff
@@ -14,8 +14,9 @@ export const messageErrorHandler = (error: any) => {
     }
 
     if (
-        error.message === 'Not authorized to publish' ||
-        error.message === 'Not authorized to subscribe'
+        error.message === 'Unauthorized to publish this message.' ||
```

Comment:
longer term you might want to consider an enum for error messages

Reply:
i'll define proper error types for the next pr.